packages/aardvark-shared/src/models.ts

```diff
@@ -29,3 +29,5 @@ export const g_builtinModelSphere = g_builtinModelPrefix + "sphere/sphere.glb";
 export const g_builtinModelStar = g_builtinModelPrefix + "star.glb";
 export const g_builtinModelTrashcan = g_builtinModelPrefix + "trashcan.glb";
 export const g_builtinModelWindowIcon = g_builtinModelPrefix + "window_icon.glb";
+export const g_builtinModelAutoLaunchActive = g_builtinModelPrefix + "rocketship.glb";
+export const g_builtinModelAutoLaunchInactive = g_builtinModelPrefix + "trashcan.glb";
```

Comment:
I'm a little torn about these names. The other names are generally descriptive of what the models are, and then the places where they're used pick one of those. On the other hand, there are a few places where the recycle icon and the star icon are used to mean "delete" and "favorite". 

Maybe do both? Make a model constant named "rocketship" and then a constant called AutoLaunchActive that's value is just the Rocketship constant? (And the same thing for the inactive icon.)

Reply:
Changed them to gadget constants and left models descriptive of the content, didn't notice that theme.